api/v1alpha1/condition_consts.go

```diff
@@ -37,3 +37,23 @@ const (
 	// DeletedReason (Severity=Info) documents a condition not in Status=True because the underlying object was deleted.
 	DeletedReason = "Deleted"
 )
+
+// Conditions related to the VirtualMachineImages
+const(
+	// VirtualMachineImageOsTypeSupportedCondition denotes that the OS type in the virtual machine image object is
```

Comment:
Here and below:
https://github.com/golang/go/wiki/CodeReviewComments#initialisms
`OsType`-> `OSType`

Also, `virtual machine image object` -> `VirtualMachineImage object`